gradio/events.py

```diff
@@ -516,6 +531,10 @@ class Events:
         "load",
         doc="This listener is triggered when the {{ component }} initially loads in the browser.",
     )
+    key_up = EventListener(
+        "key_up",
+        doc="This listener is triggered when the user presses a key while the {{ component }} is focused.",
+    )
```

Comment:
I think `keypress` is a little more intuitive, partly because we might want to change precisely _when_ this even is fired (keypresses have multiple stages) but also because 'press' is a little more generic. key press phases are pretty close to an implementation detail.